### PR TITLE
Add REVERTED state to TransactionStates.

### PR DESCRIPTION
**Pull request by liayoo**
Related: https://github.com/ainblockchain/ain-blockchain/issues/879

### CODE REVIEW

File: common/constants.js [hunk 1]
```diff
@@ -556,12 +556,17 @@ const WriteDbOperations = {
  */
 const TransactionStates = {
   FINALIZED: 'FINALIZED',
+  REVERTED: 'REVERTED',
```
Comment (platfowner) [inline common/constants.js:559]: question: Is 'reverted' different from 'failed' that it's in the block while 'failed' is not?

could you add some comments for each definition?

Comment (liayoo) [reply common/constants.js]: That's correct! Added some notes. Thanks!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by liayoo** [comment]
Thanks for the review, now merging~

**Pull request merged**